jib-maven-plugin/CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ All notable changes to this project will be documented in this file.
 
 ### Added
 
+- <to><auth> and <from><auth> parameters with <username> and <password> fields for simple authentication, similar to the Gradle plugin ([#693](https://github.com/GoogleContainerTools/jib/issues/693))
+- Can set credentials via commandline using `jib.to.auth.username`, `jib.to.auth.password`, `jib.from.auth.username`, and `jib.from.auth.password` system properties ([#693](https://github.com/GoogleContainerTools/jib/issues/693))
```

Comment:
Q: are these system probably already work in Gradle. (But I believe Gradle users can already do this on the command line, so the system properties aren't need at all, perhaps?)

Reply:
Also, I remember there are other properties like `-Dimage=...`, which I'm gussing corresponds to this code.
```
  @Parameter(property = "image")
  private ToConfiguration to = new ToConfiguration();
```
Are they different from system properties? Can we make `jib.to.auth.username` work in this way? Although not sure which one is for which situation, honestly.

Reply:
I was trying to figure out how to do this; Q told me that the `property` field works by calling a `set()` method within `ToConfiguration`. The problem I hit is that `auth` is also within `ToConfiguration`, so I wouldn't be able to make another `set` method. So to avoid the headache I just went with system properties instead since they're set in the same way.

Reply:
I see. I am curious if @coollog knows something about it.

Reply:
I'll try messing around with it some more.